devops/playbooks/aws-ci-startup.yml

```diff
@@ -1,10 +1,39 @@
 #!/usr/bin/env ansible-playbook
 
-- name: Create CI remote hosts for testing SecureDrop config.
+- name: Provision build docker image
   hosts: localhost
   become: no
   gather_facts: true
   tasks:
+    - name: Pull in systemd enabled docker image
+      docker_image:
+        name: "{{ docker_image_sd_build }}"
+      tags: docker
+      when: "{{ docker_containers != [] }}"
+
+    - name: Start build container
+      docker_container:
+        name: "{{ item }}"
+        hostname: "{{ item }}"
+        image: "{{ docker_image_sd_build }}"
+        command: /lib/systemd/systemd
+        devices:
+          - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
+        privileged: true
```

Comment:
Assuming your image was built from https://github.com/freedomofpress/docker-debian-systemd, you shouldn't need this (unless there's some reason other than systemd that you're specifying it).